react/components/ProductImages/components/Video/Vimeo.js

```diff
@@ -64,18 +64,18 @@ class Vimeo extends Component {
 
   render() {
     const { iframe } = this.state
-    const { className, id } = this.props
+    const { className, id, cssHandles } = this.props
 
     this.props.playing
       ? this.executeCommand('play')()
       : this.executeCommand('pause')()
 
     return (
-      <div style={iframe.divStyle} className={`relative ${className}`}>
+      <div style={iframe.divStyle} className={`relative ${className} ${cssHandles.iframeContainer}`}>
         <iframe
           ref={this.iframeRef}
           title={id}
-          className="absolute top-0 left-0 w-100 h-100"
+          className={`${cssHandles.iframe} absolute top-0 left-0 w-100 h-100`}
```

Comment:
What's the difference between `cssHandles.iframe` and `cssHandles.video`? And of `cssHandles.iframeContainer` and `cssHandles.videoContainer`? If they're supposed to be different, these names don't say much of what is what 🤔

Reply:
Oh, this should be video too, I thought I changed it, thx